epub33/rs/index.html

```diff
@@ -591,25 +594,25 @@ <h3>Spine</h3>
 						independently bookmarked or annotated).</span>
 					<span id="confreq-rs-pkg-duplicate-item-hyperlink" data-tests="#pkg-spine-duplicate-item-hyperlink"
 						>When a Reading System follows a hyperlink to a resource referenced multiple times in the spine,
-						the Reading System MUST move to the position of the first occurrence of the document in the
-						linear reading order.</span>
+						the Reading System MUST navigate to the first occurrence of the document in the linear reading
+						order.</span>
 				</p>
 
 				<section id="spine-overrides">
 					<h4>Spine Overrides</h4>
 
-					<p>When a spine <code>itemref</code> element's <code>properties</code> attribute contains an
-						override of a <a data-cite="epub-33#app-rendering-vocab">global rendering property</a>
-						[[EPUB-33]], Reading Systems MUST follow the requirements for the override's global value when
-						displaying that spine item.</p>
+					<p>When a spine <code>itemref</code> element's <code>properties</code> attribute overrides a
+						<a href="https://www.w3.org/TR/epub-33/#app-rendering-vocab">global rendering property</a>
```

Comment:
We use the 'data-cite' idiom for cross links; better keep it for the sake of consistency.

Reply:
```suggestion
						<a data-cite="epub-33#app-rendering-vocab">global rendering property</a>
```